README.md

```
@@ -320,10 +192,86 @@ done
 1. Go to [Circle's Faucet](https://faucet.circle.com)
-2. Select Base Sepolia
-3. Choose USDC
+2. Select **Base Sepolia**
+3. Choose **USDC**
 4. Enter your wallet address
 
+## How x402 Works
+
+The x402 protocol enables payment-gated APIs using HTTP 402 responses and gasless EIP-3009 token transfers.
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)